README.md

```diff
@@ -1,2 +1,48 @@
-# Newman-Conway-Sequence
-Project for the CS Fun Algorithms Lesson
+# Dynamic Programming
+
+Dynamic programming is a strategy for developing an algorithm where each subproblem is solved and the results recorded for use in solving larger problems.  In this exercise you will write a pair of dynamic programming methods.
+
+## Newman-Conway Sequence
+
+[Newman-Conway sequence] is the one which generates the following integer sequence.  1 1 2 2 3 4 4 4 5 6 7 7….. and follows below recursive formula.
```

Comment:
Did you intend for this to be a link?

Reply:
Nope but it makes sense for it to be, thank you for the callout!